data_trust_logger/health_audit/health_auditor.py

```diff
@@ -0,0 +1,37 @@
+import json
+from time import sleep
+import os
+
+from threading import Thread
+
+from data_trust_logger.health_audit.data_resources_collector import \
+    instantiate_data_resources_collector
+from data_trust_logger.health_audit.mci_collector import \
+    instantiate_mci_collector
+
+class HealthAuditor(Thread):
+    def __init__(self):
+        Thread.__init__(self)
+
+    def audit(self):
+        while True:
+            mci_collector = instantiate_mci_collector()
+            mci_metrics = mci_collector.collect_metrics()
+            
+            data_resources_collector = instantiate_data_resources_collector()
+            data_resources_metrics = data_resources_collector.collect_metrics()
+
+            metrics_blob = {
+                "mci_metrics" : mci_metrics,
+                "data_resources_metrics": data_resources_metrics
+            }
+
+            this_directory = os.path.abspath(os.path.dirname(__file__))
+            with open(f'{this_directory}/metrics_blob.json', 'w') as f:
```

Comment:
Same comment here. Write it to /tmp. In thinking about this, there may be a chance that you'd run into a race condition when the file is being written to and read from at the same time. One way to mitigate this risk is to first write the output to a temporary file (e.g. metrics_blob.json.tmp) and then replace the metrics_blob.json file with the temp file.

Reply:
Yes, that absolutely seems like a realistic scenario. I'll make that change.